lib/dockerregistry/inventory.go

```diff
@@ -1953,13 +1957,12 @@ func MKPopulateRequestsForPromotionEdges(
 
 // RunChecks runs defined PreChecks in order to check the promotion.
 func (sc *SyncContext) RunChecks(
-	edges map[PromotionEdge]interface{},
```

Comment:
I didn't check directly, but can you make sure that these diffs that are not related to your image size check are in a separate commit?

Reply:
Ping?